src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -256,16 +264,13 @@ def run_sequential_stages(self, checkpoint: Optional[str] = None):
                 self.train(checkpoint=checkpoint, stage=stage_name)
             checkpoint = None
 
-            if (
-                self._training_args.output_dir
-                != TrainingArguments.__dataclass_fields__["output_dir"].default
-            ):
+            if self._training_args.output_dir != DEFAULT_OUTPUT_DIR:
```

Comment:
I know that this is unrelated to this PR, but holy crap does this seem like a dumb line. If I'm reading it correctly, if the user passes in exactly output_dir="./output", the model just won't save???

Reply:
I've confirmed my suspicions. This is deeply horrifying.

Reply:
The best solution would to be set `output_dir` default to `None`, then populate it at the Trainer/Oneshot. Unfortunately because strings are primitives in python, we cannot differentiate between a user passed string and a default string of the same value using `is`.

Reply:
This is something we decided on as group. We need a good argument to make changes. 
Not saving was already considered.

Reply:
We can set a bool before parsing it into dataclass, we pop it anyway. 
use that bool to be used for saving logic.

Reply:
I’ve definitely ran models and been able to save without specifying the output dir as exactly that when running finetune? Or is this different

Reply:
The parser is a shared method, so should be the same.
Maybe in finetune --output_dir was set?

Reply:
Probably bc output is set to "./output" in training args as default and it is used in trainer. So training logic using that is probably being saved. 

So if output dir is not set 
Then in outshot -> not save
If in fine tune -> always save